modules/git/tree_entry.go

```diff
@@ -179,3 +181,30 @@ func (tes Entries) Sort() {
 func (tes Entries) CustomSort(cmp func(s1, s2 string) bool) {
 	sort.Sort(customSortableEntries{cmp, tes})
 }
+
```

Comment:
Adding a note for other reviewers: this file doesn't differentiate between go-git and non-go-git impl, so it won't need multiple implementations based on which backend is used.

Reply:
Total side-tangent, but does anyone use go-git?

Reply:
go-git version has been used in almost all Windows users.